grails-app/controllers/org/pih/warehouse/reporting/ReportController.groovy

```diff
@@ -408,7 +408,7 @@ class ReportController {
                     sw.append("Tags").append(",")
 
                     command.locations?.each { location ->
-                        sw.append("QoH ").append(location?.name).append(",")
+                        sw.append("QoH " + (location?.name).toString().replace(',','')).append(",")
```

Comment:
Nope. Please don't do this. There are tons of examples of how to do this properly see ```StringEscapeUtils.escapeCsv()```. And please don't use the addition operator here, use append like we had before. One last note, let's use display the location code if it exists 
```
location?.locationCode?:location?.name
```

Reply:
@jmiranda I've tried StringEscapeUtils.escapeCsv() before but it didn't work. It just added quotes but there were still extra columns after each comma.

Reply:
@DominikaHatala fixed it. seems like ```StringEscapeUtils.escapeCsv()``` 
 was working fine, but since it wasn't wrapping the "QoH " part of the string it was causing the issue you discovered. I just removed "QoH " because it's somewhat unnecessary. If we need to add it back we should just add it to the initialization of String locationName.